ui/spells/spells.html

```diff
@@ -60,7 +60,7 @@
       <div class="skillTP-grid">TP</div>
       <div class="skillCP-grid">CP</div>
       <div class="skillexpertise-grid">Exper.</div>
-      <div class="skillattribute-grid">IQ ±</div>
+      <div class="skillattribute-grid arcane-iq-plus-minus-attribute">IQ ±</div>
```

Comment:
This is specifically one column that I want to not word wrap. No visual change with this delta.
![image](https://user-images.githubusercontent.com/5629800/210296263-67116fb3-164f-43ed-9cb5-327eded7e35f.png)